packages/postgresql_otel/manifest.yml

```
@@ -0,0 +1,29 @@
+format_version: 3.1.3
+name: postgresql_otel
+title: "PostreSQL OpenTelemetry Assets"
+version: 0.1.0
+source:
+  license: "Elastic-2.0"
+description: "PostgreSQL OpenTelemetry Assets"
```

```suggestion
description: "PostgreSQL Assets for OpenTelemetry Collector"
```
Just a suggestion to not duplicate the title in the description